.github/workflows/weekly.yml

```diff
@@ -13,9 +13,11 @@ jobs:
       matrix:
         include:
           - dependencies: black python3-isort
-            task: fmt-travis
+            task: make -f Makefile fmt-travis
           - dependencies: yamllint
-            task: yamllint
+            task: make -f Makefile yamllint
+          - dependencies: pylint python3-pyparsing python3-hs-dbus-signature
```

Comment:
Remove unused dependency here as well. Dependencies should be the same between the nightly and the CI tasks.